.github/workflows/continuous-integration-and-delivery.yml

```diff
@@ -39,7 +45,7 @@ jobs:
     needs: test
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2.3.4
```

Comment:
@carryall I think we need to add it, in v2 , it cant push to heroku with out it. https://mikecoutermarsh.com/github-actions-deploy-to-heroku/#comment-4880683814
```yml
with:
  fetch-depth: 0
```

Reply:
✅  added in [ac7ff3b](https://github.com/carryall/survey-challenge/pull/2/commits/ac7ff3b296db91fb312c8783918cf13dfa2a565e)